maintaining-and-tuning/monitoring.md

```diff
@@ -153,3 +153,25 @@ access to it can read aggregated metrics about your GRR installation. With these
 about the number of workers, flow activity, and service health can be derived. Make sure to limit
 access to the port, for example by employing a firewall. Furthermore, read
 [Prometheus Security](https://prometheus.io/docs/operating/security/).
+
+## Example visualization setup
+This example will walk you through setting up Grafana as a dedicated visualization software to parse, display, query and set up alerts for the metrics scraped by Prometheus. These instructions assume that GRR server and Prometheus are both up and running.
+
+1. [Install Grafana](https://grafana.com/docs/grafana/latest/installation/#install-grafana) as suggested to your operating system.
+
+2. After Grafana is installed, you may [start the Grafana server](https://grafana.com/docs/grafana/latest/installation/debian/#2-start-the-server) by executing in a terminal (assuming your operating system either Debian or Ubuntu and you installed the latest OSS release):
+```
+sudo systemctl daemon-reload
+sudo systemctl start grafana-server
+sudo systemctl status grafana-server
+```
+This will get the Grafana server up and running on `http://<host>:3000` (usually, `<host>=localhost`).
+
+3. Grafana is now set up and can be visited at `http://<host>:3000`. The username and password should be "admin"; please change it.
+If the Grafana UI doesn't show up, either the [installation of Grafana](https://grafana.com/docs/grafana/latest/installation/#install-grafana) or the [server run](https://grafana.com/docs/grafana/latest/installation/debian/#2-start-the-server) failed; make sure to check the official documentation.
+
+4. [Set up Prometheus as a data source](https://grafana.com/docs/grafana/latest/features/datasources/prometheus/#prometheus-data-source) for Grafana, so that Grafana can display the metrics from Prometheus. To do that, follow the guide [here](https://grafana.com/docs/grafana/latest/features/datasources/add-a-data-source/#add-a-data-source).
+
+5. Grafana is set up and ready to show metrics scraped by Prometheus. You can start by either [creating your own dashboards](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#create-a-dashboard) or [importing exisiting dashboards](https://grafana.com/docs/grafana/latest/reference/export_import/#importing-a-dashboard) into Grafana.
```

Comment:
Would it be possible to provide commands or steps necessary to import GRR-provided dashboards into Grafana?

Reply:
At the moment, the sample GRR dashboards are not in google/grr; should I add them?

Reply:
Yes, please create a PR for that. 
I'd probably create the following folder structure:

monitoring/prometheus for what you have in 
https://github.com/tsehori/grr/tree/master/prometheus

and monitoring/grafana for grafana dashboards.

Reply:
Ok so before I commit changes to the doc regarding the Grafana alerting, I will first commit the Grafana dashboards that include the sample alerts to google/grr/prometheus and then submit a commit here with all relevant changes.

Reply:
SGTM!

Reply:
I'd prefer to move "prometheus" folder one level deep to "monitoring" ("monitoring" folder has to be created). And then put "grafana" into "monitoring" too.
Basically, putting both "grafana" and "prometheus" at the root level feels like polluting the root namespace too much.